boost-common/src/main/java/io/openliberty/boost/common/utils/BoostUtil.java

```diff
@@ -75,5 +78,38 @@ public static void extract(File artifact, File projectDirectory, String dependen
     public static String makeVariable(String propertyName) {
         return "${" + propertyName + "}";
     }
+    
+    public static String encrypt(String libertyInstallPath, String password) throws IOException {
+    	
+    	Runtime rt = Runtime.getRuntime();
+    	String[] commands = { libertyInstallPath + "/bin/securityUtility", "encode", password };
+    	Process proc = rt.exec(commands);
```

Comment:
Is there an API equivalent or do we need a new process?

Reply:
There is, but I think it's only available if you are running in a Liberty server. I'm not sure how to access it from Boost code.